can-stache-converters.js

```diff
@@ -69,13 +70,26 @@ stache.registerConverter("either-or", {
 });
 
 stache.registerConverter("equal", {
-	get: function(compute, comparer){
-		var val = (compute && compute.isComputed) ? compute() : compute;
-		return val === comparer;
+	get: function(){
+		var args = makeArray(arguments);
+		if (args.length > 1) {
+			var comparer = args.pop();
+
+			return args.every(function(compute) {
```

Comment:
is `.every` in every browser ... hahaha ... I kill myself ... but seriously ... it is in IE9 so yay!